systemd/ceph-rbd-mirror@.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Ceph rbd mirror daemon
+After=network-online.target local-fs.target
+Wants=network-online.target local-fs.target
+
+[Service]
+LimitNOFILE=1048576
+LimitNPROC=1048576
+EnvironmentFile=-/etc/sysconfig/ceph
+Environment=CLUSTER=ceph
+ExecStart=/usr/bin/rbd-mirror -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
```

Comment:
By default, if not specified, librados will read the configuration file whose name matches the cluster: http://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#running-multiple-clusters